synthetics/test_schema.go

```diff
@@ -166,133 +173,133 @@ func makeTestIPSchema(mode schemaMode) *schema.Schema {
 }
 
 func makeTestAgentSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"target": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 	})
 }
 
 func makeTestFlowSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"target": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 		"target_refresh_interval_millis": {
 			Type:     schema.TypeInt,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 		"max_tasks": {
 			Type:     schema.TypeInt,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 		"type": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
```

Comment:
AFAIK, type is required for flow tests.

Reply:
Addressed: a2e73bcd0db6697b5392fdb69a572f74228bc594

Reply:
Done